frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -231,6 +233,7 @@ function ToolRadioButton({
     <RadioButtonWithTooltip
       title={`${name} – ${description}`}
       disabledTitle={`${name} – ${disabledExplanation}`}
+      onOpenChange={onOpenChange}
```

Comment:
I think, the two line changes in `ToolRadioButton` shouldn't be necessary or are they?

Reply:
Yes, you are right, sensei.